### PR TITLE
netkvm: RxCapacity changes

### PR DESCRIPTION
**Pull request by kfir-manor**
test patches to check if RxCapacity changes pass HCK-CI

### CODE REVIEW

File: NetKVM/netkvm-base.txt [hunk 1]
```diff
@@ -267,6 +269,8 @@ String_128 = "128"
 String_256 = "256"
 String_512 = "512"
 String_1024 = "1024"
+String_1024 = "2048"
```
Comment (YanVugenfirer) [inline NetKVM/netkvm-base.txt:272]: This is an error, it should be:
String_2048 and next line String_4096

**Comment by kostyanf14** [comment]
NICStrictPropertyValidation fail is ok (need to rebuild image)

NDISTest 6.0 - [1 Machine] - 1c_WMICoverage fail:

**Comment by YanVugenfirer** [comment]
@kostyanf14  can you please check 1c_WMICoverage on master? There was a massive playlist update last week.

**Comment by YanVugenfirer** [comment]
@ybendito actually, it is our WMI info that fails the test: 234E1FBF-37DC-4882-B01E-18F47CC0A40E

https://github.com/virtio-win/kvm-guest-drivers-windows/blob/fa5d4d67047fc3d0d52f7e7e793dba8a96bf37d7/NetKVM/Common/netkvm.mof#L7

**Comment by YanVugenfirer** [comment]
WMI test failed because of outdated filter.

**Review by YanVugenfirer — APPROVED** [review]

**Pull request merged**